test/unit/bodyCullTest.js

```diff
@@ -133,14 +166,14 @@ describe(`cullBodies`, () => {
     await cullBodies()
     {
       const transaction = await TransactionModel.findById(tranCulled._id)
-      should(transaction.request.body).undefined()
-      should(transaction.response.body).undefined()
+      should(transaction.request.bodyId).undefined()
+      should(transaction.response.bodyId).undefined()
     }
 
     {
       const transaction = await TransactionModel.findById(tranLeftAlone._id)
-      should(transaction.request.body).eql('test')
-      should(transaction.response.body).eql('test')
+      should(transaction.request.bodyId).eql(requestBodyId)
+      // should(transaction.response.bodyId).eql(responseBodyId)
```

Comment:
@BMartinos I'm currently working in these tests - can I uncomment these 3 assertions

Reply:
@MattyJ007 Yeah you can uncomment those :+1: . This should have been undone with this initial work, but got over-looked.